src/components/screens/Bridge/useBridge.composable.js

```diff
@@ -0,0 +1,20 @@
+import { coreStore } from '@/store';
+import pkg from '../../../../package.json';
```

Comment:
why not from root? you can add alias in config, f.e. `@@` is a root, when `@` is  a src

Reply:
updated